library/src/webapp/js/spinner.js

```diff
@@ -203,7 +206,7 @@ SPNR.disableInputs = function( clickedElement, escapeList )
 SPNR.disableButtons = function( clickedElement, escapeList )
 {
     // Get all the button elements
-    const buttons = SPNR.nodeListToArray( document.getElementsByTagName( "button" ) );
+    const buttons = content.document.getElementsByTagName( "button" );
```

Comment:
I think this should just be content.getElementsByTagName rather than content.document.getElementsByTagName

```suggestion
    const buttons = content.getElementsByTagName( "button" );
```